mule-migration-tool-api/src/test/java/com/mulesoft/tools/migration/project/model/pom/PomModelTestCase.java

```diff
@@ -225,5 +266,36 @@ public void setDescription() {
 
       assertThat("Description should exist on the pom model", model.getDescription(), equalTo(value));
     }
+
+    @Test
+    public void setParentGroupId() {
+      model.getParent().setGroupId("new-group-id");
+      assertThat("GroupId is not the expected", model.getParent().getGroupId(), equalTo("new-group-id"));
+    }
+
+    @Test
+    public void setParentArtifactId() {
+      model.getParent().setArtifactId("new-artifact-id");
+      assertThat("GroupId is not the expected", model.getParent().getArtifactId(), equalTo("new-artifact-id"));
+    }
+
+    @Test
+    public void setParentVersion() {
+      model.getParent().setVersion("2.0.0");
+      assertThat("GroupId is not the expected", model.getParent().getVersion(), equalTo("2.0.0"));
+    }
+
+    @Test
+    public void setParentRelativePath() {
+      model.getParent().setRelativePath("new-path");
+      assertThat("RelativePath is not the expected", model.getParent().getRelativePath(), equalTo("new-path"));
+    }
+
+    @Test
+    public void setParent() {
+      Parent parent = new Parent();
+      model.setParent(parent);
+      assertNotNull(model.getParent());
```

Comment:
Assert that parent it's actually modified. e.g
```
@Test
public void setParent() {
  Parent parent = new Parent();
  parent.setGroupId("myGroupId");
  parent.setArtifactId("myArtifactId");
  parent.setVersion("1.0.0");
  model.setParent(parent);
  assertNotNull(model.getParent());
  assertThat("Parent groupId is not the expected", model.getParent().getGroupId(), equalTo("myGroupId"));
  assertThat("Parent artifactId is not the expected", model.getParent().getArtifactId(), equalTo("myArtifactId"));
  assertThat("Parent version is not the expected", model.getParent().getVersion(), equalTo("1.0.0"));
}
```
Using a parent builder class this construction can be simplified
```
@Test
public void setParent() {
  Parent parent =
      new Parent.ParentBuilder().withGroupId("myGroupId").withArtifactId("myArtifactId").withVersion("1.0.0").build();
  model.setParent(parent);
  assertNotNull(model.getParent());
  assertThat("Parent groupId is not the expected", model.getParent().getGroupId(), equalTo("myGroupId"));
  assertThat("Parent artifactId is not the expected", model.getParent().getArtifactId(), equalTo("myArtifactId"));
  assertThat("Parent version is not the expected", model.getParent().getVersion(), equalTo("1.0.0"));
}
```

Reply:
A builder for a wrapper is not needed !?! 

Why create Builder with 100 lines of. The code have also to be tested.

Keep it easy and write code that is really needed.